### PR TITLE
Update request approval time

### PR DESCRIPTION
**Pull request by khoaguin**
## Description
Update `approval_time` of a request after it is approved according to [this ticket](https://github.com/OpenMined/Heartbeat/issues/375)

### CODE REVIEW

File: packages/syft/src/syft/service/request/request.py [hunk 1]
```diff
@@ -206,6 +206,7 @@ def apply(self, context: AuthedServiceContext) -> Result[SyftSuccess, SyftError]
             change_status.applied = True
             self.history.append(change_status)
 
+        self.approval_time = DateTime.now()
```
Comment (rasswanth-s) [inline packages/syft/src/syft/service/request/request.py:209]: MAYBE: By the PR: https://github.com/OpenMined/PySyft/pull/7742 we also have the notion of denying a request.

We could modify the variable name to be `updated_at` to capture approval and denial?
As the approval time , might be fixed only to approval
cc : @shubham3121  , @IrinaMBejan

Comment (khoaguin) [reply packages/syft/src/syft/service/request/request.py:209]: I talked to @shubham3121 and we will change `approval_time` to `updated_at`

**Review by shubham3121 — APPROVED** [review]

**Comment by shubham3121** [comment]
@khoaguin can you fix linting for this PR.

**Comment by khoaguin** [comment]
Sure @shubham3121 I am looking into it

**Pull request merged**